python-stdlib/logging/logging.py

```diff
@@ -202,8 +202,8 @@ def critical(msg, *args):
     getLogger().critical(msg, *args)
 
 
-def exception(msg, *args):
-    getLogger().exception(msg, *args)
+def exception(msg, *args, **kwargs):
```

Comment:
Would it be better to do:
```py
def exception(msg, *args, exc_info=True):
   getLogger().exception(msg, *args, exc_info)
```
?

That way it's clear what args are supported.  In particular, general keyword args are **not** supported, only `exc_info` is.

Reply:
Good idea; I've updated it!